theano/scan_module/scan_utils.py

```diff
@@ -185,6 +185,8 @@ def clone(output,
         as the original graph. If False, clone them. Note that cloned
         shared variables still use the same underlying storage, so they
         will always have the same value.
+
+    :param copy_inputs: deprecated, use share_inputs.
```

Comment:
This parameter apear in the auto generated doc in the function signature. When I saw it there, I was wondering why it was there and not documented. So this make it clear to people they should ignore it from the function signature.